### DISCUSSION TITLE
Update Feed in Multiple Marketplaces at the same time !!

### DISCUSSION THREAD

**Discussion by ghost** [abuzuhri/Amazon-SP-API-CSharp#91] [Q&A]
_Hi. First of all, very nice library.
I wanted to ask, how can I upload a different feed files in multiple Marketplaces at the same time ?

Thank you_

I found it. Thank you !!

**Comment by abuzuhri** [comment]
HI @qemajlosmanihub 

I don't think you can upload diffent feed in same request , you need to submit may request .

/Tareq

**Reply by ghost** [reply]
Thanks, but what about this ?
Does this uploads Feed in DE and also in ES now ??

            var test_feed = amapi.Feed.SubmitFeed(feed_content
            , FeedType.POST_FLAT_FILE_PRICEANDQUANTITYONLY_UPDATE_DATA
            , new List<string>() { "DE", "ES" } //Marketplace
            , null
            , ContentType.TXT
            );

**Reply by abuzuhri** [reply]
Yes , same feed can submit to many market place but not different feeds in same request
